frontend/src/components/Forms/Login/Login.tsx

```diff
@@ -132,11 +131,11 @@ function Login() {
           <a className={classes.link} onClick={() => navigate("/register", {replace: true})}>
             Register
           </a>
-          <br /> or play as a <a className={classes.link}>guest</a>
+          <br /> or play as a <a className={classes.link} onClick={() => navigate("/gamemodes", {replace: false})}>guest</a>
```

Comment:
replace is false by default I think (btw I didn't understand this before if you set replace to true it replaces the current route on the history stack - so if replace was true and you went from login to game modes it would only appear in your history as if you had visited game modes. by making it false you are able to press the back button in your browser to go back to login if you choose)

Reply:
✔️